src/test/utils/testUtils.ts

```diff
@@ -179,4 +179,19 @@ export async function stopLibertyserver() {
   (await input).click();
   await delay(10000);
 }
+
+export async function clearCommandPalette() {
+  await new Workbench().executeCommand('Clear Command History');
+  await delay(30000);  
+  const dialog = new ModalDialog();
+  const message = await dialog.getMessage();
+
+  expect(message).contains('Do you want to clear the history of recently used commands?');
+  const details = await dialog.getDetails();
+
+  expect(details).equals(`This action is irreversible!`);
```

Comment:
No need to check for details. message title and button check is enough.

Reply:
Removed the check for details of dialog